frontend/src/scenes/Auth/components/SignUp/styles.module.scss

```diff
@@ -7,6 +8,11 @@
   margin: 0 auto;
   border-radius: 6px;
   background-color: white;
+  height: 100vh;
+  overflow:auto;
+  -ms-overflow-style: none;
+  overflow: -moz-scrollbars-none;
+  &::-webkit-scrollbar { width: 0; }
```

Comment:
Why do we need these changes?  I think @thkrp has already fixed an issue with the sign-up form height

Reply:
OK.  Since NotFound is not needed here, I can temporarily remove it.  When adding routes for password recovery, you will have to go back.  The routing file takes an unusual approach, namely Routing without a Switch.  When using NotFound, I could see both NotFound and the authentication page.  Using Switch solved this problem.  But there was a problem with the height of the registration form.

Reply:
You can keep Switch here, it's correct to have it there

Reply:
Sorry.  I did not quite understand.  Leave only Switch or styles and Switch.

Reply:
only Switch

Reply:
Ok.